tcms/bugs/tests/test_views.py

```diff
@@ -68,3 +81,58 @@ def test_reopen_a_closed_bug(self):
         self.assertRedirects(response, redirect_url)
         bug.refresh_from_db()
         self.assertTrue(bug.status)
+
+
+class TestNewBug(LoggedInTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        user_should_have_perm(cls.tester, 'bugs.add_bug')
+
+        cls.url = reverse('bugs-new')
+
+        cls.summary = 'A shiny new bug!'
+        cls.product = ProductFactory()
+        cls.version = VersionFactory(product=cls.product)
+        cls.build = BuildFactory(product=cls.product)
+        cls.post_data = {
+            'summary': cls.summary,
+            'reporter': cls.tester.pk,
+            'product': cls.product.pk,
+            'version': cls.version.pk,
+            'build': cls.version.pk,
+        }
+
+    def test_get_view(self):
+        response = self.client.get(self.url)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertIn('page_title', response.context)
+        self.assertIn('form_post_url', response.context)
+        self.assertEqual(
+            response.context['form'].initial['reporter'].pk, self.tester.pk
+        )
+        self.assertTemplateUsed(response, 'bugs/mutable.html')
+
+    def test_new_bug(self):
+        initial_bug_count = Bug.objects.count()
+        resp = self.client.post(self.url, self.post_data, follow=True)
+
+        self.assertEqual(resp.status_code, 200)
+        self.assertEqual(Bug.objects.count(), initial_bug_count + 1)
+
+    def test_new_bug_comment(self):
+        self.client.post(self.url, self.post_data, follow=True)
+
+        bug_created = Bug.objects.last()
+        comment_query = Comment.objects.filter(object_pk=bug_created.pk, user=self.tester)
+        self.assertEqual(comment_query.count(), 1)
```

Comment:
This scenario is out of context. The referenced bug relates to tests only for the New view.